RiscVPlatformPkg/Universal/Sec/SecMain.c

```diff
@@ -514,10 +580,13 @@ VOID EFIAPI SecCoreStartUpWithStack(UINTN hartid, struct sbi_scratch *scratch)
   HartsIn ++;
   for (;;);
 #endif
-  if (hartid == FixedPcdGet32(PcdBootHartId)) {
-    sbi_console_init(scratch); // Initial OpenSBI internal serial console on boot Hart.
-    sbi_ecall_init(); // Initial ecall registration in SEC phase for handling further traps.
-    PeiCore();
+  if (HartId == FixedPcdGet32(PcdBootHartId)) {
+    // Doesn't work when I put an argument here :/ With 1 as arg I get an access fault

```

Comment:
cc @changab

Reply:
are you sure the exception happens right after "scratch->next_arg1 = (UINTN)1;", or it happens later?
So, actually we can set Scratch->next_addr to PeiCore() in this file, without giving arg1 a value.

Reply:
No, it happens somewhere during `sbi_init`.
Yes, we don't need to pass anything else, so I'll just remove this.